paying-for-college/disclosures/models.py

```diff
@@ -10,6 +10,12 @@ class School(models.Model):
     data_json = models.TextField()
     city = models.CharField(max_length=50)
     state = models.CharField(max_length=2)
+    accreditor = models.CharField(max_length=255, blank=True)
+    url = models.TextField(blank=True)
+    degrees_predominant = models.TextField(blank=True)
+    degrees_highest = models.TextField(blank=True)
+    operating = models.BooleanField(default=True)
+    KBYOSS = models.BooleanField(default=False)  # shopping-sheet participant
```

Comment:
This is super, outrageously nit-picky, but I don't think I'd know (in fact, I don't), what `KBYOSS` means. Perhaps expand a bit on the comment?